prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -232,6 +247,10 @@ void rxr_pkt_handle_ctrl_sent(struct rxr_ep *rxr_ep, struct rxr_pkt_entry *pkt_e
 	case RXR_COMPARE_RTA_PKT:
 		rxr_pkt_handle_rta_sent(rxr_ep, pkt_entry);
 		break;
+	case RXR_DC_EAGER_MSGRTM_PKT:
+	case RXR_DC_EAGER_TAGRTM_PKT:
+		/* no action to be taken here */
+		break;
```

Comment:
Do not we need handle ```RXR_DC_EAGER_MSGRTM_PKT``` and ```RXR_DC_EAGER_TAGRTM_PKT``` in ```rxr_pkt_handle_send_completion```?

Reply:
I do not think so. The only thing we do for `RXR_EAGER_MSGRTM_PKT` and `RXR_EAGER_TAGRTM_PKT` when calling `rxr_pkt_handle_send_completion` is writing completion to the TX/RX queue by calling `rxr_cq_handle_tx_completion`. And this is something we should do when receiving a receipt packet when the sender requests `FI_DELIVERY_COMPLETE`

Reply:
I see, can you add this to the comments.

Reply:
Sure.